doc/source/conf.py

```diff
@@ -432,14 +432,11 @@ def setup(app):
 # -----------------------------------------------------------------------------
 # Coverage checker
 # -----------------------------------------------------------------------------
-coverage_ignore_modules = r"""
-    """.split()
-coverage_ignore_functions = r"""
-    test($|_) (some|all)true bitwise_not cumproduct pkgload
-    generic\.
-    """.split()
-coverage_ignore_classes = r"""
-    """.split()
+coverage_ignore_modules = []
```

Comment:
This is strange. Was the intent really to create an empty array? Why use `r"""    """.split()` instead of an empty array?

Reply:
That code is from 2008 :) I suspect it was left that way as a sort of template for future additions. Why not use a list for the others? The only thing I can think of is that it was easy to do, but who knows. The change looks fine.